cypress/integration/Crosscutting/LanguageToggleTest/index.js

```diff
@@ -0,0 +1,33 @@
+/// <reference types="Cypress" />
+
+import { Given, Then, And, When } from "cypress-cucumber-preprocessor/steps";
+
+Given('user is navigating to {string}', (a) => {
+    cy.visit(a);
+    Cypress.on('uncaught:exception', (err, runnable) => {
+        // we expect a 3rd party library error with message 'Cannot read property'
+        // and don't want to fail the test so we return false
+        if (err.message.includes('Cannot read property')) {
```

Comment:
Same comment as in another pr - please include the entire console error message to the check

Reply:
updated the error